kolibri/content/models.py

```diff
@@ -320,7 +320,6 @@ class Exam(models.Model):
         can_be_deleted_by=(),
     )
 
-    id = UUIDField(primary_key=True)
```

Comment:
@jamalex: "We can leave this field in for now, to minimize breakage. `id`s will still be `UUIDField`s when Morango is integrated".